sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/JobLifecycleSamples.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Azure.DigitalTwins.Core;
+using Azure.DigitalTwins.Core.Samples;
+using static Azure.DigitalTwins.Core.Samples.SampleLogger;
+using static Azure.DigitalTwins.Core.Samples.UniqueIdHelper;
+
+namespace Azure.DigitalTwins.Samples
+{
+    internal class JobLifecycleSamples
+    {
+        /// <summary>
+        /// Creates a new model with a random Id
+        /// Decommission the newly created model and check for success
+        /// </summary>
+        public async Task RunSamplesAsync(DigitalTwinsClient client, Options options)
+        {
+            PrintHeader("JOB LIFECYCLE SAMPLE");
+
+            // For the purpose of this example we will create an import job to import models, twins, and relationships.
+            // We have to make sure the job Id is unique within the DT instance.
+
+            string sampleImportJobId = await GetUniqueJobIdAsync(SamplesConstants.TemporaryJobPrefix, client);
+
+            // upload a sample file to their storage blob? or should we require that they already have one uploaded and provide the name of the file as input?
```

Comment:
I'm still not sure how we should do this. Options are:

**1. We upload input blob to customer storage account**
Pros: Less work for the customer, seems in line with the rest of the samples for not making the customer come up with any data (e.g model names, twin names, etc).
Cons: More work, more permissions needed to get the sample running.

**2. Customer uploads input blob to storage account**
Pros: Less work for us, keeps the sample scoped to modifying adt instance only (not customer storage blob)
Cons: More work for customer to get sample up and running, could be malformed.

Reply:
Discussed offline with @sjiherzig and we will go with option 1, plus we will add some setup code to create the storage account for the customer if possible.

Reply:
Opting not to create the storage account since it will require extra setup of permissions etc that may be complicated. Instead, will upload the input blob to the container as part of the sample which will be used in the import api sample.